packages/react-native/scripts/common.js

```diff
@@ -69,7 +69,7 @@ function resolveAddonFile(addon, file, extensions = ['js', 'mjs', 'ts']) {
 
   for (const ext of extensions) {
     try {
-      const filePath = path.join(addon, `${file}.${ext}`);
+      const filePath = `${addon}/${file}.${ext}`;
```

Comment:
looks janky but I guess it should work.

Reply:
😅 yeah I know but atleast I know what its doing haha